index.php

```diff
@@ -104,15 +106,16 @@
     // New filter functionality, searching and listing of requests.
     echo $renderer->render_index_search_controls($context, $categoryid, $courseid, $stateid, $baseurl, $search, $faculty);
 
-    if (capability_checker::can_export_csv()) {
+    // Here we use preference, not capability. 
```

Comment:
I don't think this comment is not needed, it seems clear from the code that you are reading a preference.